package.json

```diff
@@ -11,6 +11,7 @@
     "build:content": "node --max-old-space-size=8192 script/build-content.js",
     "build:content:test": "node script/test-cms-export.js",
     "build:webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/webpack.config.js",
+    "build:webpack:prod": "NODE_ENV=production yarn build:webpack --env.buildtype vagovprod",
```

Comment:
This is intended to be temporary. I just added it so I don't mistakenly run `yarn build` and end up with a `build/localhost` folder.

Reply:
looks good